KsApi/models/UserCreditCard.swift

```diff
@@ -0,0 +1,19 @@
+public struct GraphUserCreditCard: Swift.Decodable {
+
+  public private(set) var storedCards: CreditCardConnection
+
+  public struct CreditCard: Swift.Decodable, Equatable {
+    public private(set) var expirationDate: String
+    public private(set) var id: String
+    public private(set) var lastFour: String
+    public private(set) var type: String
+
+    public var formattedExpirationDate: String {
+      return String(expirationDate.dropLast(3))
```

Comment:
Hmm I'm not sure we should be doing date formatting in this way, if something changes with the order of how the date is returned from the API, this could break. The safest way to do this would be to convert the String we get into a Date, and then use a `DateFormatter` to format the date into a correctly formatted string.

Reply:
Oh now I see it's formatted in the view model. But is it necessary to truncate the string?

Reply:
Before implementing this, I confirmed with the API that the format wouldn't change. The `dropLast` function is just getting rid of the day returned by graphQL. The cards expiration date usually has the format `MM/yyyy` and the day returned is always `01`, making the expiration date wrong.